### DISCUSSION TITLE
Flashlist typescript ?

### DISCUSSION THREAD

**Discussion by shop-fluencer** [Shopify/flash-list#578] [Q&A]
Hello,

Whats the ListRenderItem prop in flashlist?

```
  const renderItem: ListRenderItem<ICard> = ({ item }) => (
    <Card
      {...item}
    />
  )
```

**Comment by fortmarek** [comment]
Hey!

It's just a type encapsulating the [callback mentioned in the docs](https://shopify.github.io/flash-list/docs/usage#renderitem).
